Libraries/RoslynTripleSlash/TripleSlashSyntaxRewriter.cs

```diff
@@ -0,0 +1,409 @@
+            return GetXmlTrivia("typeparam", attribute, contents, leadingWhitespace);
+        }
+
+        private SyntaxTriviaList GetReturns(string text, SyntaxTriviaList leadingWhitespace)
+        {
+            // For when returns is empty because the method returns void
+            if (string.IsNullOrWhiteSpace(text))
+            {
+                return new();
+            }
+
+            SyntaxList<XmlNodeSyntax> contents = GetContentsInRows(text);
+            XmlElementSyntax element = SyntaxFactory.XmlReturnsElement(contents);
+            return GetXmlTrivia(element, leadingWhitespace);
+        }
+
+        private SyntaxTriviaList GetExceptions(DocsMember member, SyntaxTriviaList leadingWhitespace)
+        {
+            SyntaxTriviaList exceptions = new();
+            // No need to add exceptions in secondary files
+            if (!UseBoilerplate && member.Exceptions.Any())
+            {
+                foreach (SyntaxTriviaList exceptionsTrivia in member.Exceptions.Select(
+                    exception => GetException(exception.Cref, exception.Value, leadingWhitespace)))
+                {
+                    exceptions = exceptions.AddRange(exceptionsTrivia);
+                }
+            }
+            return exceptions;
+        }
+
+        private SyntaxTriviaList GetException(string cref, string text, SyntaxTriviaList leadingWhitespace)
+        {
+            TypeCrefSyntax crefSyntax = SyntaxFactory.TypeCref(SyntaxFactory.ParseTypeName(cref));
+            XmlTextSyntax contents = SyntaxFactory.XmlText(GetTextAsTokens(text, leadingWhitespace));
+            XmlElementSyntax element = SyntaxFactory.XmlExceptionElement(crefSyntax, contents);
+            return GetXmlTrivia(element, leadingWhitespace);
+        }
+
+        private SyntaxTriviaList GetSeeAlsos(DocsMember member, SyntaxTriviaList leadingWhitespace)
+        {
+            SyntaxTriviaList seealsos = new();
+            // No need to add exceptions in secondary files
+            if (!UseBoilerplate && member.SeeAlsos.Any())
+            {
+                foreach (SyntaxTriviaList seealsoTrivia in member.SeeAlsos.Select(
+                    s => GetSeeAlso(s.Cref, leadingWhitespace)))
+                {
+                    seealsos = seealsos.AddRange(seealsoTrivia);
+                }
+            }
+            return seealsos;
+        }
+
+        private SyntaxTriviaList GetSeeAlso(string cref, SyntaxTriviaList leadingWhitespace)
+        {
+            TypeCrefSyntax crefSyntax = SyntaxFactory.TypeCref(SyntaxFactory.ParseTypeName(cref));
+            XmlEmptyElementSyntax element = SyntaxFactory.XmlSeeAlsoElement(crefSyntax);
+            return GetXmlTrivia(element, leadingWhitespace);
+        }
+
+        private SyntaxTokenList GetTextAsTokens(string text, SyntaxTriviaList leadingWhitespace)
+        {
+            string whitespace = leadingWhitespace.ToFullString().Replace(Environment.NewLine, "");
```

Comment:
This was a pain to figure out. I do not know how to programatically detect the indentation automatically and add it as a prefix where needed. For now, this works.